packages/nimble-components/src/rich-text-mention/base/models/mention-validator.ts

```diff
@@ -22,10 +22,10 @@ export const baseValidityFlagNames = [
 export class RichTextMentionValidator<
     ValidityFlagNames extends readonly string[] = typeof baseValidityFlagNames
 > extends Validator<typeof baseValidityFlagNames | ValidityFlagNames> {
-    public constructor(
-        private readonly mentionInternals: MentionInternals,
-        configValidityKeys: ValidityFlagNames
-    ) {
+    @observable
+    public isRichTextMentionValid = true;
```

Comment:
What is `isRichTextMentionValid` for (given we already have `isValid()`)? Do we need this as an observable?

Reply:
That's a good point. I had added this here as a replacement to the `validConfiguration` observable in `MentionInternals`, but I never found anything that was listening to changes to that observable that needed to be updated. Do you know why `validConfiguration` was needed on `MentionInternals`?

Reply:
Removed.